src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/relevantremand/service/FindHistoricReleaseDateService.kt

```diff
@@ -83,13 +83,13 @@ class FindHistoricReleaseDateService(
     val latestRelease = releaseDates.maxOrNull()
     if (latestRelease != null) {
       if (latestRelease.isBefore(calculateAt)) {
-        throw UnsupportedCalculationException("The release date is before the calculation date.")
+        throw UnsupportedCalculationException("The release date $latestRelease, from calculations $calcluationIds is before the calculation date $calculationDate.")
```

Comment:
Spelling of `calcluationIds` here should be `calculationIds` (likely a problem on a previous line)